Specs/Scene/ModelExperimental/CustomShaderSpec.js

```diff
@@ -220,6 +220,17 @@ describe("Scene/ModelExperimental/CustomShader", function () {
         scene.destroyForSpecs();
       });
 
+      var shaders = [];
+      afterEach(function () {
+        for (var i = 0; i < shaders.length; i++) {
+          var shader = shaders[i];
+          if (!shader.isDestroyed()) {
+            shader.destroy();
+          }
+          shaders.length = 0;
```

Comment:
Should `shaders.length = 0;` be outside the loop? Same comment for `textureManagers` below.